### PR TITLE
Ready events

### PR DESCRIPTION
**Pull request by VirgilChindris**
Added a call to overlay.element.onDrawingDone() in drawOverlay function.
Converted onDrawingDone function to event.
Related PR: https://github.com/ni-kismet/webcharts-legends/pull/86

### CODE REVIEW

File: jquery.flot.js [hunk 1]
```diff
@@ -2604,18 +2604,22 @@ Licensed under the MIT license.
...
-        function drawOverlay() {
+        function drawOverlay(plot) {
             redrawTimeout = null;
 
             if (!octx) {
                 return;
             }
             overlay.clear();
             executeHooks(hooks.drawOverlay, [octx, overlay]);
+            var event = new CustomEvent('onDrawingDone');
```
Comment (cristiingineru) [inline jquery.flot.js]: Please, make sure the new event is mentioned in the documentation: https://github.com/ni-kismet/engineering-flot/blob/728b73fcd7411911877a1827aa31d271680e74f0/API.md#plot-methods in the **triggerRedrawOverlay** section.

Maybe here as well: https://github.com/ni-kismet/engineering-flot/blob/728b73fcd7411911877a1827aa31d271680e74f0/API.md#hooks in the **drawOverlay** section.

\+ specify this small API enhancement in the CHANGELOG.md file.

File: API.md [hunk 2]
```diff
@@ -1184,6 +1184,12 @@ can call:
     immediately, instead a timer is set to catch multiple successive
     redraws (e.g. from a mousemove). You can get to the overlay by
     setting up a drawOverlay hook.
+    This function will also trigger the creation and dispatching the
```
Comment (ralucaportase) [inline API.md]: of the*

File: API.md [hunk 3]
```diff
@@ -1642,6 +1648,12 @@ hooks in the plugins bundled with Flot.
     metrics computed by Flot, e.g. plot.width()/plot.height(). See the
     crosshair plugin for an example.
 
+    This hook creates and dispatches the 'onDrawingDone' custom event.
```
Comment (cristiingineru) [inline API.md]: Maybe dispatches only, in order to be dispatched it has to be created anyway.

File: CHANGELOG.md [hunk 4]
```diff
@@ -1,6 +1,10 @@
 # Changelog
 All notable changes to this project will be documented in this file.
 
+## [1.1.1]
```
Comment (cristiingineru) [inline CHANGELOG.md]: The 1.1.0 was not released yet, please upend this change to the 1.1.0 section.

**Pull request merged**